### DISCUSSION TITLE
How to deploy this theme through github pages?

### DISCUSSION THREAD

**Discussion by MarkRushB** [Sped0n/bridget#304] [General]
I want to deploy this theme through github pages, however I found there is no index.html in the root directory, so wondering how to do that?

**Comment by 713b76b7-2247-4bd5-98a7-014226363f9b** [comment]
I'm not very familiar with GitHub Pages, but perhaps you could try using GitHub Actions instead?

https://github.com/peaceiris/actions-gh-pages

**Reply by MarkRushB** [reply]
Thanks, I figured it out and finally deploy it to vercel!

BTW, how can I config the homepage? right now when I click the Bridget at the bottom left corner, it will show 404 not found

**Reply by 713b76b7-2247-4bd5-98a7-014226363f9b** [reply]
Have you setup `baseURL` in [config](https://github.com/Sped0n/bridget/blob/e82fe6cab2669bde3f27493704b5847464651747/exampleSite/config/_default/hugo.toml#L4)?

By default the homepage link is `/`.
![SCR-20240512-nyud](https://github.com/Sped0n/bridget/assets/70063177/2b1166d5-dbb0-454a-84f5-2c79737b603a)

**Reply by MarkRushB** [reply]
Thank you sooo much! I really appreciate you for contributing this project, this is awesome!!! I like it so much!

BTW, I still have a question, for the website on the mobile device, is that effect expected? On the mobile device, looks like the behavior is not same as PC's, mobile's is more like as you swipe up, photos appear from below and stack on top of each other. Is that expected?

**Reply by 713b76b7-2247-4bd5-98a7-014226363f9b** [reply]
> Is that expected?

Yes, it is intended.
